internal/commands/key.go

```diff
@@ -0,0 +1,171 @@
+	Read                  key.Binding
+	Favourite             key.Binding
+	ToggleReads           key.Binding
+	ToggleFavourites      key.Binding
+	Refresh               key.Binding
+	OpenInBrowser         key.Binding
+	oQuit                 key.Binding
+	oClearFilter          key.Binding
+	oCancelWhileFiltering key.Binding
+	oNextPage             key.Binding
+	oPrevPage             key.Binding
+}
+
+// ViewportKeyMapT shows *all* keybinds, pulling from viewport.DefaultKeyMap()
+type ViewportKeyMapT struct {
+	Quit          key.Binding
+	Escape        key.Binding
+	OpenInBrowser key.Binding
+	GotoStart     key.Binding
+	GotoEnd       key.Binding
+	Next          key.Binding
+	Prev          key.Binding
+	ShowFullHelp  key.Binding
+	CloseFullHelp key.Binding
+}
+
+// ListKeyMap shows either (o)verrides or new keybinds
+var ListKeyMap = ListKeyMapT{
+	Open: key.NewBinding(
+		key.WithKeys("enter"),
+		key.WithHelp("enter", "open"),
+	),
+	Favourite: key.NewBinding(
+		key.WithKeys("f"),
+		key.WithHelp("f", "favourite"),
+	),
+	ToggleFavourites: key.NewBinding(
+		key.WithKeys("F"),
+		key.WithHelp("F", "toggle show favourite"),
+	),
+	Read: key.NewBinding(
+		key.WithKeys("m"),
+		key.WithHelp("m", "mark read"),
+	),
+	ToggleReads: key.NewBinding(
+		key.WithKeys("M"),
+		key.WithHelp("M", "toggle show read"),
+	),
+	Refresh: key.NewBinding(
+		key.WithKeys("r"),
+		key.WithHelp("r", "refresh"),
+	),
+	OpenInBrowser: key.NewBinding(
+		key.WithKeys("o"),
+		key.WithHelp("o", "open in browser"),
+	),
+	// o for override
+	oQuit: key.NewBinding(
+		key.WithKeys("ctrl+c"),
+		key.WithHelp("ctrl+c", "quit"),
+	),
+	oClearFilter: key.NewBinding(
+		key.WithKeys("esc", "q"),
+		key.WithHelp("esc/q", "clear filter"),
+	),
+	oCancelWhileFiltering: key.NewBinding(
+		key.WithKeys("esc"),
+		key.WithHelp("esc", "cancel"),
+	),
+	oPrevPage: key.NewBinding(
+		key.WithKeys("left", "h", "pgup"),
+		key.WithHelp("←/h/pgup", "prev page"),
+	),
+	oNextPage: key.NewBinding(
+		key.WithKeys("right", "l", "pgdown"),
+		key.WithHelp("→/l/pgdn", "next page"),
+	),
+}
+
+// ViewportKeyMapT shows *all* keybinds, pulling from viewport.DefaultKeyMap()
+var ViewportKeyMap = ViewportKeyMapT{
+	Next: key.NewBinding(
+		key.WithKeys("l", "right"),
+		key.WithHelp("l/→", "next"),
+	),
+	Prev: key.NewBinding(
+		key.WithKeys("h", "left"),
+		key.WithHelp("h/←", "prev"),
+	),
+	Quit: key.NewBinding(
+		key.WithKeys("ctrl+c"),
+		key.WithHelp("ctrl+c", "quit"),
+	),
+	Escape: key.NewBinding(
+		key.WithKeys("esc", "q"),
+		key.WithHelp("q/esc", "escape"),
+	),
+	OpenInBrowser: key.NewBinding(
+		key.WithKeys("o"),
+		key.WithHelp("o", "open in browser"),
+	),
+	GotoStart: key.NewBinding(
+		key.WithKeys("g", "home"),
+		key.WithHelp("g", "top"),
+	),
+	GotoEnd: key.NewBinding(
+		key.WithKeys("G", "end"),
+		key.WithHelp("G", "bottom"),
+	),
+	ShowFullHelp: key.NewBinding(
+		key.WithKeys("?"),
+		key.WithHelp("?", "more"),
+	),
+	CloseFullHelp: key.NewBinding(
```

Comment:
Don't think this is working as expected, I see `? more` on initial load, but then see `? close help` when the help is open.